modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/frontend/data/set/filter/DisplayDateRangeFDSFilter.java

```diff
@@ -54,7 +54,7 @@ public DateFDSFilterItem getMaxDateFDSFilterItem() {
 
 		return new DateFDSFilterItem(
 			calendar.get(Calendar.DAY_OF_MONTH),
-			calendar.get(Calendar.MONTH) + 1, calendar.get(Calendar.YEAR));
+			calendar.get(Calendar.MONTH) + 1, calendar.get(Calendar.YEAR) + 10);
```

Comment:
@oliv-yu You can pass `0` in place of the three values (day of month, month and year) to indicate that there's no upper limit.